IronCoreLabs/ironoxide-java
# [#103] Add support for building an android .aar package

See #103 

To test
* install cross with 'cargo install cross --version 0.1.16`
* in `ironoxide-java/android`, run `build.sh`

If successful, there should be an output aar in `android/ironoxide-android/build/outputs/aar`
